tests/dragonfly/cluster_test.py

```diff
@@ -1320,6 +1320,74 @@ async def list_counter(key, client: aioredis.RedisCluster):
     )
 
 
+@dfly_args({"proactor_threads": 4, "cluster_mode": "yes"})
+async def test_cluster_config_reapply(df_local_factory: DflyInstanceFactory):
+    """Check data migration from one node to another."""
```

Comment:
The entire setup here (~40 lines) is identical to below
maybe we can have a test fixture (is that how it's called?) with setup functions and config apply etc that could reduce the noise here?

Reply:
I'm not sure what it should look like. I would prefer to have the result as soon as possible and then do tests refactoring